typingdna_activelock/README.md

```diff
@@ -0,0 +1,69 @@
+
+# TypingDNA ActiveLock
+
+## Overview
+
+[TypingDNA ActiveLock][3] is a Continuous Endpoint Authentication app that helps prevent unauthorized access to your company computers. Once installed on a user's PC, it continuously verifies the user by the way they type. If an unauthorized typing pattern is detected, ActiveLock instantly locks the computer and/or logs the data to your desired logging platform (i.e. Datadog).
+
+To visualize your data in Datadog, a custom ActiveLock app will need to be configured and installed. Note: This will be the same install for all of your company computers.
+
+Configuration is straightforward and takes few minutes, please follow the steps in the **Configure** tab to get started.
+
+## Setup
+
+### Configuration
+
+**I. Generate a Datadog API key**
+1. Within your Datadog account, navigate to [Organization settings > API keys][4].
+2. Generate a new "API key".
+
+**II. Get your custom install app.**
+1. Navigate to [this custom install form][7] and submit your newly generated API key and your Datadog Region (e.g. US1, EU) along with your company details.
+2. Once we receive your information, we'll send you a custom ActiveLock app that you'll have to install on your company computers. We'll send any further details over email.
+3. After installation, logs should start appearing in [Log explorer][5].
+
+**III. Set up log features (facets/measures).**
+1. Once logs flow into Datadog, Navigate to [Log explorer][5].
+2. On the left side panel, select the "+Add" button to add the following facets/measures.
+Note: Facets and measures are case sensitive. Please include `@` in the path definition.
+
+|Path| Facet/Measure |Type|Display name|
+|--|--|--|--|
+|`@username`|facet|string|username|
+|`@installID`|facet|string|installID|
+|`@main_app`|facet|string|main_app|
+|`@result`|facet|string|result|
+|`@keyboard_id`|facet|string|keyboard_id|
+|`@version`|facet|string|version|
+| `@score` | measure |integer|score|
+| `@speed` | measure |integer|speed|
+| `@training_strength` | measure |integer|training_strength|
+
+
+### Validation
+
+To view your ActiveLock logs in Datadog, navigate to [Log explorer][5] and set `source:typingdna_activelock`
+
+To open the dashboard, navigate to [All Dashboards][6] where you should find the **TypingDNA ActiveLock** dashboard.
+
+
+## Data Collected
+
+### Log collection
+
+TypingDNA ActiveLock logs are collected and sent to Datadog directly from each app.
+
+In order to be able to view the logs correctly in the TypingDNA ActiveLock dashboard you will need to set the facets and measures exactly as in the Configuration steps.
```

Comment:
```suggestion
TypingDNA ActiveLock logs are collected and sent to Datadog directly from each application.

In order to view the logs correctly in the TypingDNA ActiveLock dashboard, you need to set the facets and measures in the [Configuration](#configuration) section.

### Metrics

The TypingDNA ActiveLock integration does not include any metrics.

### Events

The TypingDNA ActiveLock integration does not include any events.

### Service Checks

The TypingDNA ActiveLock integration does not include any service checks.

```

Reply:
emarsha94 suggested to delete those 3 sentences before now you're asking me to add them back in, IMO you guys need to make sure you suggest the same things